Tests/Unit/InstanceTests.swift

```diff
@@ -80,4 +80,162 @@ class InstanceTests: XCTestCase {
     let isDifferent = dcOne !== dcTwo
     XCTAssertTrue(isDifferent)
   }
+
+  struct VariableTypeOne: OperationVariable {
```

Comment:
Were you going to add a `VariableTypeTwo` and add a test for variables of different types?

Reply:
Yes I was going to. Thanks for pointing it out. I've added a couple more tests with different types.